src/detectors/FHCAL/FHCAL.cc

```diff
@@ -69,7 +69,7 @@ extern "C" {
           "HcalEndcapPInsertMergedHits", {"HcalEndcapPInsertRecHits"}, {"HcalEndcapPInsertMergedHits"},
           {
             .readout = "HcalEndcapPInsertHits",
-            .fields = {"layer", "slice"},
+            .fields = {"layer", "slice", "side"},
             .refs = {1, 0},
```

Comment:
Should `refs` have the same length as `fields`?

Reply:
i don't know what refs does.  What does it do?

Reply:
now that I look at this more carefully, I am not sure if this is necessary for what I am doing, so I will remove this particular change and keep the other ones in this pull request